docs/setting-up-fuzzing/build_afl.bash

```diff
@@ -36,15 +36,15 @@ declare -a afl_sources=(
 )
 for source_file in "${afl_sources[@]}"
 do
-  curl -O "https://cs.chromium.org/codesearch/f/chromium/src/third_party/afl/src/$source_file"
+  curl -H "cookie:REDIRECT_STATUS=optout" -O "https://cs.chromium.org/codesearch/f/chromium/src/third_party/afl/src/$source_file"
```

Comment:
Instead, can you update the link to the new location.

Reply:
Would it be better to use google/AFL GitHub repo instead?

Reply:
Can you provide the link that points to the new raw location at CodeSearchV2?

The 302 is incorrect to get the raw file. It wants to point to https://source.chromium.org/chromium/chromium/src/+/master:third_party/afl/src/afl-fuzz.c

which means to get the raw I'd have to go to the gitiles version and append format TEXT https://chromium.googlesource.com/chromium/src/+/master/third_party/afl/src/afl-fuzz.c?format=TEXT then base64 decode the response to get the file.

The reason why I chose the cookie was because CodeSearchV2 doesn't offer a nice way to download the file without base64 decoding it first. https://github.com/google/gitiles/issues/106.

Using google AFL repo it would be https://raw.githubusercontent.com/google/AFL/master/afl-fuzz.c

So the switch would be something like

curl -I "https://raw.githubusercontent.com/google/AFL/master/$source_file"

Reply:
https://raw.githubusercontent.com/google/AFL/* links SGTM, no need to use chromium ones.

Reply:
Github changed as the source and I added "android-ashmem.h" to the list of the files as there was an update https://github.com/google/AFL/pull/46. I also chose the Github LLVM project for the AFL driver cpp file.